modular_zubbers/code/modules/reagents/chemistry/reagents/other.dm

```diff
@@ -0,0 +1,9 @@
+
+//Splash-based recipe to turn improvised gauze into sterile gauze.
+/datum/reagent/space_cleaner/sterilizine/expose_obj(obj/O, reac_volume)
```

Comment:
rename O to "sprayed"/"splashed", something like that.

Reply:
Is there a reason this is necessary? Using O as the name for an object is the standard way of naming single objs as targets, while M is the standard for mobs.